bftengine/tests/testMsgsCertificate/CMakeLists.txt

```diff
@@ -3,6 +3,12 @@ find_package(GTest REQUIRED)
 add_executable(msgsCertificate_test msgsCertificate_test.cpp $<TARGET_OBJECTS:logging_dev>)
 add_test(msgsCertificate_test msgsCertificate_test)
 
+target_include_directories(msgsCertificate_test
+        PRIVATE
+        ${bftengine_SOURCE_DIR}/src/preprocessor
```

Comment:
why do you need the preprocessor as a dependency ?

Reply:
In order to use 'ClientReplyMsg' used in my test, I have needed to include '${bftengine_SOURCE_DIR}/src/preprocessor'.

Reply:
can't you use another message, which is not in the pre-processor?

Reply:
Correction:
It was in order to use 'SimpleClientImp' and the method implemented there.

Reply:
I don't get it... SimpleClientImp is not defined in the preprocessor.

Reply:
SimpleClientImp include "messages/ClientPreProcessRequestMsg.hpp".
My class use SimpleClientImp and this dir is unknown for my class, that's why I added it.

Reply:
Alright, I see - thanks for clarifying.